### PR TITLE
Feature/calculate cube

### LINKED ISSUE #10: Prepare the endpoint to sum the area/volume (3-4)

### PR DESCRIPTION
**Pull request by grazik**
* implement getCube method for Building ald Level Class
* add proper endpoint to return cube of given location

Resolves #10

### CODE REVIEW

**Comment by mmargooo** [comment]
It looks fine. I think it's almost the same as sum-the-area feature so nothing should go wrong :)

File: src/main/java/pl/put/poznan/buildingInfo/model/Building.java [hunk 1]
```diff
@@ -48,9 +48,8 @@ public Float getArea() {
 
     public Float getCube() {
         // calculate cubage of all levels
-
-        // return temporary value in order to compile code properly
-        return (float) 0.0;
+
```
Comment (roertbb) [inline src/main/java/pl/put/poznan/buildingInfo/model/Building.java]: unnecessary empty line ;)

File: src/main/java/pl/put/poznan/buildingInfo/model/Level.java [hunk 2]
```diff
@@ -45,9 +45,8 @@ public Float getArea() {
 
     public Float getCube() {
         // calculate cubage of all levels
-
-        // return temporary value in order to compile code properly
-        return (float) 0.0;
+
```
Comment (roertbb) [inline src/main/java/pl/put/poznan/buildingInfo/model/Level.java]: unnecessary empty line ;)

Comment (grazik) [reply src/main/java/pl/put/poznan/buildingInfo/model/Level.java]: I think, it's done now ;)

File: src/main/java/pl/put/poznan/buildingInfo/model/Building.java (hunk 1 above)
Comment (grazik) [reply src/main/java/pl/put/poznan/buildingInfo/model/Building.java]: I think, it's done now ;)

**Pull request merged**